.github/workflows/cml-report.yml

```diff
@@ -0,0 +1,63 @@
+name: Generate CML Report
+on:
+  pull_request:
+    # types: [ labeled ]
+    branches:
+      - master
+
+jobs:
+  generate_cml_report:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    container: 
+      image: ghcr.io/moja-global/rest_api_gcbm:master
+      ports:
+        - "8080:8080"
+    steps:
+    
+      - name: Checkout repository
+        uses: actions/checkout@v3
+          
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          
+      - name: Run the Simulation
+        run: |
+          apt-get update && apt-get install unzip -y
+          pushd local/rest_api_gcbm/tests/
+          unzip linux-demo.zip
+          cd config/
+          /opt/gcbm/moja.cli --config_file gcbm_config.cfg --config_provider provider_config.json
+          popd
+          cd local/rest_api_gcbm && mkdir GCBM.CompileResults
+          
+      - name: Checkout the Repository

```

Comment:
I'd be happy to add this dependency to the `flint.gcbm` image.

Reply:
Will create this as a good first issue for someone to work on 👍